doc/source/data/overview.rst

```diff
@@ -23,6 +23,10 @@ It provides flexible and performant APIs for distributed data processing:
 
 Ray Data is built on top of Ray, so it scales effectively to large clusters and offers scheduling support for both CPU and GPU resources. Ray Data uses `streaming execution <https://www.anyscale.com/blog/streaming-distributed-execution-across-cpus-and-gpus>`__ to efficiently process large datasets.
 
+.. note::
+
+    Ray Data isn't a replacement for generic ETL pipelines like Spark or distributed
+    DataFrame libraries like Dask.
```

Comment:
I would reword this a bit with our most recent positioning. 

"Ray Data does not have a SQL interface and is not meant as a replacement for generic ETL pipelines like Spark."

I would remove the mention of Dask.